### DISCUSSION TITLE
地图移动，放大，缩小都会导致cpu暴涨，从而出现严重卡顿现象

### DISCUSSION THREAD

**Discussion by lyric-zemin** [antvis/L7#2260] [Q&A]
https://github.com/antvis/L7/assets/54665054/967ab9b1-5133-4d12-b66a-0d7d8f29526d

**Comment by lzxue** [comment]
可以看看使用 Map 地图有没有这个问题

**Reply by lyric-zemin** [reply]
也是这样

**Reply by lyric-zemin** [reply]
不过在关闭控制台的情况下，Map卡顿明显改善，GaodeMap仍然很卡顿
